readme.md

```diff
@@ -119,33 +161,56 @@ What is the difference between the two following lines of code?
 @artist.save!
 ```
 
-> Your answer...
+Clark Answer: Methods with and ! attached to the end of them usually mean that they will
+modify the object they are calling on. Save! will raise an error if not successful. Save
+will return true or false
 
```

Comment:
:+1:

Pretty much. In this case, however, the `!` refers to the error raising - not modifying the object (but that is correct in other scenarios).